components/cart.css

```diff
@@ -0,0 +1,65 @@
+/* main container styles  */
+.main-container {
+  color: var(--FONT-COLOR);
+  padding: 4rem;
+}
+
+.main-container > h2 {
+  text-align: center;
+  padding: 2rem;
+}
+
+/* product card styles  */
+.card-product {
+  max-width: 35rem;
+  overflow: hidden;
+  background: var(--ZINC-COLOR-LIGHT);
+  border-radius: 8px;
+}
+
+.card-product-image {
+  max-width: 15rem;
+  object-fit: cover;
+}
+
+.card-product-details {
+  padding: 2rem;
+  width: 20rem;
+}
+
+.old-price {
+  margin: 0.5rem;
```

Comment:
You can use px (upto 10px), if units are less than 1rem.

Reply:
updated as per suggestion.